runtime/domain.c

```diff
@@ -750,11 +748,6 @@ static void* backup_thread_func(void* v)
 
   caml_domain_set_name("Backup");
 
-  CAML_EVENTLOG_IS_BACKUP_THREAD();
-
-  /* TODO: how does the backup thread interact with the eventlog infra?
```

Comment:
I had a ponder about this when I was removing that section and I'm not sure there is. You will see runtime events from the backup threads but you'll never see them interleaved with those from the main thread because of the runtime lock.